specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json

```diff
@@ -239,6 +239,15 @@
               "name": "SearchOptions"
             }
           },
+          {
+            "name": "semanticConfiguration",
+            "in": "query",
+            "type": "string",
+            "description": "The name of a semantic configuration that will be used when processing documents for queries of type semantic.",
```

Comment:
What do you think about "re-ranking documents" instead of "processing documents"??

Reply:
The only trouble with using "re-ranking" is that it omits the semantic captions/answers that this is also being used for. 

I'm wondering if we should be more explicit; something like:  "The name of the semantic configuration that lists which fields should be used for semantic ranking, captions, highlights, and answers."

Reply:
+1 on Derek's suggested phrase